specifyweb/specify/migration_utils/default_cots.py

```
@@ -136,22 +136,25 @@ def fix_tectonic_unit_treedef_discipline_links(apps):
-            empty_tectonic_unit_treedef.discipline = empty_disciplines.last()
-            empty_tectonic_unit_treedef.save()
+    empty_tectonic_unit_treedefs = list(
+        Tectonicunittreedef.objects.filter(discipline__isnull=True)
+    )
+    empty_disciplines = list(
+        Discipline.objects.filter(tectonicunittreedef__isnull=True)
+    )
+
+    for discipline, tectonic_unit_treedef in zip(
+        empty_disciplines, empty_tectonic_unit_treedefs
+    ):
```

Ruff (0.15.13)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)